src/iexamine.cpp

```diff
@@ -56,37 +56,31 @@ void iexamine::gaspump(player *p, map *m, int examx, int examy)
     add_msg(m_info, _("Out of order."));
 }
 
-void iexamine::atm(player *p, map *m, int examx, int examy)
+void iexamine::atm(player *const p, map *, int , int )
 {
-    (void)m; //unused
-    (void)examx; //unused
-    (void)examy; //unused
-    int choice = -1;
-    const int purchase_cash_card = 1;
-    const int deposit_money = 2;
-    const int withdraw_money = 3;
-    const int transfer_money = 4;
-    const int transfer_all_money = 5;
-    const int cancel = 0;
-    long amount = 0;
-    long max = 0;
-    std::string popupmsg;
-    int pos;
-    int pos2;
-    item *dep;
-    item *with;
+    enum : int { // menu choices
+        cancel, purchase_cash_card, deposit_money,
+        withdraw_money, transfer_money, transfer_all_money
+    };
+
+    constexpr int card_cost      = 100;
```

Comment:
Did you even bother to check? card_cost and move_cost_bulk are only used one place each.